molecule/test12/converge.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Converge
+  hosts: all
+  tasks:
+    - name: "Include ansible-role-openhpc"
```

Comment:
Needs fixing in the same way as https://github.com/stackhpc/ansible-role-openhpc/commit/b167d4b7015caba149d1716d8ce9272dd1034893